packages/ui/src/components/Designer/RightSidebar/ListView/SelectableSortableContainer.tsx

```diff
@@ -136,27 +146,17 @@ const SelectableSortableContainer = (
         {createPortal(
           <DragOverlay adjustScale>
             {activeId ? (
-              <>
-                <ul style={{ margin: 0, padding: 0, listStyle: 'none' }}>
-                  <Item
-                    value={schemas.find((schema) => schema.id === activeId)!.key}
-                    style={{ background: token.colorPrimary }}
-                    dragOverlay
-                  />
-                </ul>
-                <ul style={{ margin: 0, padding: 0, listStyle: 'none' }}>
-                  {selectedSchemas
-                    .filter((item) => item.id !== activeId)
-                    .map((item) => (
-                      <Item
-                        key={item.id}
-                        value={item.key}
-                        style={{ background: token.colorPrimary }}
-                        dragOverlay
-                      />
-                    ))}
-                </ul>
-              </>
```

Comment:
Oh, I forgot to mention, I didn't know what this was achieving so I removed it?

This seemed to create draggable list items when multiple are being moved together? (I think) but I couldn't see how this was possible. If you select multiple schemas then it always goes to the `DetailView`. I couldn't see a way to get multiple draggable schemas in the `ListView`. 

If am I wrong then I will revert this. If it's correct then there might be more code that can be removed from this file.

Reply:
I had made it so that pressing shift while selecting would allow multiple items to be moved at once, but it seems to have stopped working at some point.

It might be better not to delete it.

Reply:
OK, I will revert

Reply:
I have reverted this.